src/integrations/admin/crawl-settings-integration.php

```diff
@@ -174,9 +173,13 @@ public function add_crawl_settings_tab( $dashboard_tabs ) {
 	/**
 	 * Adds content to the Crawl Cleanup tab.
 	 *
+	 * @deprecated 20.4
+	 * @codeCoverageIgnore
+	 *
 	 * @param Yoast_Form $yform The yoast form object.
 	 */
 	public function add_crawl_settings_tab_content( $yform ) {
+		_deprecated_function( __METHOD__, 'WPSEO 20.4' );
```

Comment:
I see we're using `'Yoast SEO 20.4'` lately, even though [in our docs](https://yoast.atlassian.net/wiki/spaces/PLUGIN/pages/1763573763/How+to+deprecate+a+method+class) your form is used... 🤷

Reply:
There was a split before, but I kept with WPSEO now